util/options.cc

```diff
@@ -517,7 +517,9 @@ font_options_t::add_options (option_parser_t *parser)
     {"face-index",	0, 0, G_OPTION_ARG_INT,		&this->face_index,		"Set face index (default: 0)",		"index"},
     {"font-size",	0, default_font_size ? 0 : G_OPTION_FLAG_HIDDEN,
 			      G_OPTION_ARG_CALLBACK,	(gpointer) &parse_font_size,	font_size_text,				"1/2 numbers or 'upem'"},
-    /* TODO Add font-ppem / font-ptem. */
+    {"font-y-ppem",	0, 0, G_OPTION_ARG_INT,		&this->x_ppem,			"Set 'x pixels per EM'",		"96"},
```

Comment:
Please do it the way font-size is done. Ie. no separate X / Y option.

Also, default is NOT 96. Defaults to zero, meaning not set / size-independent.